### ISSUE TITLE
Two animationData variables in one page

### ISSUE THREAD

**Issue by kirillSkorodelov** [airbnb/lottie-web#2208]
**Windows10**

* **Browser and Browser Version: Chrome 83.0.4103.97 (Official Build) (64-bit)**
* **After Effects Version: 17.1.0**

**I've declared two animationData variables - animationData1 and animatioData2, and assigned two functions for playing it. Here's the code:
//FIRST ANIMATION AND ITS PARAMETERS
        var animationData1 ={"data exported from AE and it definetely works"};
        var params1 = {
            container: document.getElementById('lottie'),
            renderer: 'svg',
            loop: false,
            autoplay: false,
            autoloadSegments: false,
            prerender: false,
            animationData: animationData1
        };
        var template1Anim;
        template1Anim = lottie.loadAnimation(params1);
//SECOND ANIMATION AND ITS PARAMETERS
            var animationData2 ={"data exported from AE and it definetely works"}
            var params2 = {
            container: document.getElementById('lottie'),
            renderer: 'svg',
            loop: false,
            autoplay: false,
            autoloadSegments: false,
            prerender: false,
            animationData: animationData2
        };
        var template2Anim;
        template2Anim = lottie.loadAnimation(params2);

//FUNCTIONS TO ANIMATE IT IN AND OUT
        function animatTemplate1(action, line1, line2) {
            if (action == "on") {
                template1Anim.renderer.elements[5].updateDocumentData({
                    t: line1 //.toString() //.replace(/\n/g, '\r'),
                }, 0);
                template1Anim.renderer.elements[1].updateDocumentData({
                    t: arguments[2] //.toString() //.replace(/\n/g, '\r'),
                }, 0);
                template1Anim.playSegments([0, 50], true);
            }
            if (action == "off") {
                template1Anim.playSegments([190, 240], true);
            }

        };

        function animatTemplate2(action, line1) {
            if (action == "on") {
                console.log(params2)
                template2Anim.renderer.elements[0].updateDocumentData({
                    t: line1 //.toString() //.replace(/\n/g, '\r'),
                }, 0);
                template2Anim.playSegments([0, 30], true);
            }
            if (action == "off") {
                template2Anim.playSegments([64, 82], true);
            }

        };

**

**I expected it both work when you send a command like animatTemplate1("on", "text1", "text2) and   animatTemplate2("on", "text1"). **

**What actually happened? Please include as much _relevant_ detail as possible. Actually only the first command triggers the animation and it's perfect, no problem at all. But the second animation doesn't play and doesn't actually returns any error. Separately second animation reacts to these commands. If I remove first set of animation data - var animationData1- it works fine. Looks like the first variable with animation data blocks the second or smth like that. **

**Please provide a download link to the After Effects file that demonstrates the problem.**

**Comment by bodymovin** [comment]
hi, are you loading both animations on the same container?
Can you provide a link with a working example?

**Comment by kirillSkorodelov** [comment]
yes, to the same container. Tried to use different ones, no luck although. I'll try to publish an example on codepen

**Comment by kirillSkorodelov** [comment]
dropped you an email with the file attached. Thanks a lot in advance!
